lib/kdump_utils.pm

```diff
@@ -37,6 +36,13 @@ sub prepare_for_kdump_sle {
         script_run 'zypper -n rr SLES-Server-Debug';
         return;
     }
+    elsif (get_var('REPO_SLE15_MODULE_BASESYSTEM_DEBUG')) {
+        my $url = "$utils::OPENQA_FTP_URL/" . get_var('REPO_SLE15_MODULE_BASESYSTEM_DEBUG');
```

Comment:
I guess we can improve here not to repeat same code, as the only difference is url. So either extract it to some function which gets url and does the thing, or combine first two if statements, define url accordingly and run same code.